Code/MinimalLib/minilib.h

```diff
@@ -99,7 +99,9 @@ class JSMol {
 
   std::string get_stereo_tags() const;
   std::string get_aromatic_form() const;
+  void set_aromatic_form();
```

Comment:
I think `convert_to_aromatic_form` (and `convert_to_kekule_form`) is a better name.

Reply:
Done.